pkg/compute/capacity/tool.go

```diff
@@ -27,28 +26,22 @@ type ToolBasedProvider struct {
 func (tool *ToolBasedProvider) GetAvailableCapacity(ctx context.Context) (models.Resources, error) {
 	toolPath, err := exec.LookPath(tool.Command)
 	if err != nil {
-		// If the tool is not installed, we can't know the number of GPUs.
-		// It is not an error to assume zero.
-		log.Ctx(ctx).Info().Msgf("cannot inspect system %s: %s not installed. %s will not be used.", tool.Provides, tool.Command, tool.Provides)
-		return models.Resources{}, nil
+		// If the tool is not installed, we can't know the available resources.
+		return models.Resources{}, errors.Wrapf(err, "tool %q is not installed or not on PATH", tool.Command)
```

Comment:
In the future it might be a good practice to be consistent around our usage of `fmt.Errorf("%w", err)` vs `errors.Wrap`. At the moment we are using both, but not a big deal.